.github/workflows/dbt_precommit_hooks.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         with:
           format: space-delimited
           token: ${{ secrets.GITHUB_TOKEN }}
+        continue-on-error: true
```

Comment:
👍 should be a warning, not break the pipeline

Reply:
I definitely thought this would just not set the FILES arg on a failure. I'm sort of shocked.